test/test-parse-module.cpp

```diff
@@ -42,11 +42,19 @@ TEST(test_parse, export_variable) {
   }
 
   {
-    spy_visitor v = parse_and_visit_statement(u8"export const x;"_sv);
+    padded_string code(u8"export const x;"_sv);
```

Comment:
Almost-blocking: Fix the input, not the assertions.
```suggestion
    padded_string code(u8"export const x = null;"_sv);
```